impl/src/main/java/io/jsonwebtoken/impl/lang/DefaultCollectionMutator.java

```diff
@@ -46,6 +47,33 @@ private boolean doAdd(E e) {
         return this.collection.add(e);
     }
 
+    public M replace(E e) {
+        if (Objects.isEmpty(e)) {
+            if (e == null)  throw new NullPointerException("Cannot be null.");
+            else            throw new IllegalArgumentException("Cannot be empty.");
+        }
```

Comment:
You can use `Assert.notEmpty()` here, see an example:
https://github.com/jwtk/jjwt/blob/508227cf4d8b59a80df2f7062406b0c47bf624e1/impl/src/main/java/io/jsonwebtoken/impl/lang/CollectionConverter.java#L60

Reply:
There is currently no overloaded `Assert.notEmpty()` for `Object`. I shall implement it.